### PR TITLE
docs: add schematics docs for organization and misc libs

### LINKED ISSUE #828: Organization and misc libs schematics docs

### PR DESCRIPTION
**Pull request by znikola**
closes #828

### CODE REVIEW

File: _pages/install/schematics.md [hunk 1]
```diff
@@ -45,15 +45,15 @@ Another example is the following, which generates an application that is ready t
...
+### Additional Commands for Core Libraries and Features
 
 By default, the `ng add @spartacus/schematics` command adds only a basic configuration of Spartacus. The following is a description of the commands you can use to extend your application:
 
 - `ng g @spartacus/schematics:add-pwa` adds a Spartacus-specific PWA module.
 - `ng g @spartacus/schematics:add-ssr` adds the SSR configuration.
 - `ng g @spartacus/schematics:add-cms-component` generates a CMS component, and adds the CMS component mapping to the specified module, or to a newly-generated module, if no module is specified. For more information, see [CMS Component Schematic](#cms-component-schematic), below.
 
-### How Spartacus Schematics Work
+## How Spartacus Schematics Work
```
Comment (znikola) [inline _pages/install/schematics.md]: I think this section should be nested under the `Adding Spartacus Core Libraries and Features to Your Angular Project` section, as it only applies to it.

File: _pages/install/schematics.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ When you run `ng add @spartacus/schematics`, the command does the following:
    - Adds server-side rendering dependencies.
    - Provides additional files that are required for SSR.
 
-### CMS Component Schematic
+## CMS Component Schematic
```
Comment (znikola) [inline _pages/install/schematics.md]: Nice, this one should be on the same level as `Adding Spartacus Core Libraries and Features to Your Angular Project` and `Installing Additional Spartacus Libraries` 👍

File: _pages/install/schematics.md (hunk 1 above)
Comment (gladius-mtl) [reply _pages/install/schematics.md]: I had thought maybe that section could stand on its own, but I see your point and have made the fix according to your suggestion 🙂

**Review by gladius-mtl — APPROVED** [review]

File: _pages/install/schematics.md (hunk 2 above)
Comment (gladius-mtl) [reply _pages/install/schematics.md]: 😅

**Pull request merged**